v2.1/export.md

```diff
@@ -0,0 +1,159 @@
+---
+title: EXPORT
+summary: Export tabular data from CockroachDB cluster in CSV format.
+toc: false
+---
+
+The `EXPORT` [statement](sql-statements.html) exports tabular data to CSV files.
+
+{{site.data.alerts.callout_danger}}The <code>EXPORT</code> feature is only available to <a href="https://www.cockroachlabs.com/product/cockroachdb/">enterprise</a> users. Also note that this feature is currently under development and is slated for full release in CockroachDB 2.1. The feature flags and behavior are subject to change. {{site.data.alerts.end}}
+
+<div id="toc"></div>
+
+## Export File Location
+
+You can use remote cloud storage (Amazon S3, Google Cloud Platform, etc.) to store the exported CSV data. Alternatively, you can use an [HTTP server](create-a-file-server.html) accessible from all nodes.
+
+For simplicity's sake, it's **strongly recommended** to use cloud/remote storage for the data you want to export. Local files are supported; however, they must be accessible identically from all nodes in the cluster.
+
+## Performance
+
+All nodes are used during tabular data conversion into CSV data, which means all nodes' CPU and RAM will be partially consumed by the [`EXPORT`](export.html) task in addition to serving normal traffic.
+
+## Cancelling Export Jobs
+
+After the export has been initiated, you can cancel it with [`CANCEL JOB`](cancel-job.html).
+
+## Synopsis
+
+{% include sql/{{ page.version.version }}/diagrams/export.html %}
+
+{{site.data.alerts.callout_info}}The <code>EXPORT</code> statement cannot be used within a <a href=transactions.html>transaction</a>.{{site.data.alerts.end}}
```

Comment:
This is true right now, but looking at it, I'm not sure why. I'll probably just send a PR to remove this check, so we could remove the warning now if we wanted, though technically it applies until next alpha. I'm fine either way.

Reply:
I vote for keeping it in till we remove it in the next alpha.